radar/radar.go

```diff
@@ -198,6 +211,27 @@ func (radar *Radar) checkLock(resourceName string) []db.NamedLock {
 	return []db.NamedLock{db.ResourceCheckingLock(resourceName)}
 }
 
+func (radar *Radar) getResourceConfig(logger lager.Logger, resourceName string) (atc.ResourceConfig, error) {
+	var found bool
+	var resourceConfig atc.ResourceConfig
+
+	config, _, err := radar.db.GetConfig()
+	if err != nil {
+		logger.Error("failed-to-get-config", err)
+		// don't propagate error; we can just retry next tick
```

Comment:
comment is no longer true, and i think we still don't want to propagate the error here